aea/cli/publish.py

```diff
@@ -112,7 +125,7 @@ def _check_is_item_in_registry_mixed(
             click.echo("Found!")
         except click.ClickException as e:
             raise click.ClickException(
-                f"Package not found neither in local nor in remote registry: {str(e)}"
+                f"Package not found neither in local nor in remote registry: {str(e)}. You can try to add {PUSH_ITEMS_FLAG}."
```

Comment:
Does it make sense?

Reply:
yes, just a hint for the user to add a flag to publish automatically